wagtailmenus/models.py

```diff
@@ -106,6 +106,12 @@ class MenuItem(models.Model):
         blank=True,
         help_text=_("Must be set if you wish to link to a custom URL."),
     )
+    link_handle = models.SlugField(
```

Comment:
Also, link_url is only named 'link_url' so that it doesn't conflict with the 'url' attribute that gets added when menu items are 'primed'. Since the handle won't be clashing with anything else, it probably makes sense to just use 'handle' as the field name, and ditch the property method.